grossmont/computer-science-and-cybersecurity-organization/pages/alumni/alumni.html

```diff
@@ -64,34 +64,36 @@ <h3>Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod temp
     <!--third member-->
     <div class="container" style="margin-top: 50px; margin-bottom: 50px;">
       <div>
-        <img class="image" ;
-          src="https://fiverr-res.cloudinary.com/images/q_auto,f_auto/gigs/173653543/original/9de3c4b3dad756c1d969fc469916f4a15bef899b/draw-bad-stick-figures-that-kinda-looks-like-you.png">
+        <img
+          src="https://media.discordapp.net/attachments/761470887001325569/1088329748943474768/IMG_2358.png?width=994&height=1092">
```

Comment:
Looks really nice! I might need someone to shoot a full body photo of me if that's the photo theme we are going with

Reply:
Doesn't have to be full body! I was hoping for something casual and not extremely formal, could be a selfie with something fun in the background or just you smiling and holding a thumbs up (though if you really wanted to, keeping your linkedin pfp would be ok!)